pkg/generator/helper/helper.go

```diff
@@ -30,11 +32,14 @@ type stepInfo struct {
 const stepGoTemplate = `package cmd
 
 import (
-	{{if .OSImport}}"os"{{end}}
```

Comment:
That changes here do not fit IMO into the scope of the PR, that makes reviews / understanding of the code harded than it needs to be.